### PR TITLE
New project created

### PR DESCRIPTION
**Pull request by BaidetskyiYurii**

### CODE REVIEW

File: NewsIOS/NewsIOS/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  NewsIOS
+//
+//  Created by Baidetskyi Jurii on 28.10.2022.
+//
+
+import UIKit
+
```
Comment (glbchk) [inline NewsIOS/NewsIOS/ViewController.swift:9]: Should we have an empty view controller? What a hell?????

**Review by glbchk — COMMENTED** [review]
Great job, but read my damn comments...

File: NewsIOS/NewsIOS/ViewController.swift (hunk 1 above)
Comment (BaidetskyiYurii) [reply NewsIOS/NewsIOS/ViewController.swift:9]: Sorry(

**Comment by BaidetskyiYurii** [comment]
All is done!

**Review by glbchk — APPROVED** [review]

**Pull request merged**